pkgs/data/fonts/alice/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, stdenv, fetchzip }:
+
+stdenv.mkDerivation (attrs: {
+  pname = "Alice";
+  version = "2.003";
+
+  outputs = [ "out" "woff2" ];
```

Comment:
Is there any reason to this multiple output here?

Reply:
See the discussion in https://github.com/NixOS/nixpkgs/issues/182465#issuecomment-1207234828: most desktop users don't need the webfonts.